client/go/outline/method_channel.go

```diff
@@ -0,0 +1,57 @@
+// Copyright 2024 The Outline Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package outline
+
+import (
+	"fmt"
+
+	"github.com/Jigsaw-Code/outline-apps/client/go/outline/platerrors"
+)
+
+// API name constants
+const (
+	// FetchResourceMethod fetches a resource located at a given URL.
+	//
+	//  - Input: the URL string of the resource to fetch
+	//  - Output: the content in raw string of the fetched resource
+	MethodFetchResource = "FetchResource"
+)
+
+// InvokeMethodResult represents the result of an InvokeMethod call.
+//
+// We use a struct instead of a tuple to preserve a strongly typed error that gobind recognizes.
+type InvokeMethodResult struct {
+	Value string
+	Error   *platerrors.PlatformError
```

Comment:
The alignment of the fields looks weird. Did `gofmt` do this?

Reply:
Fixed